src/components/ApplicationMenu/ApplicationMenu.js

```diff
@@ -21,18 +22,22 @@ import type { Task } from '../../types';
 const { app, process, Menu } = remote;
 
 type Props = {
+  selectedProject: ?string,
   selectedProjectId: ?string,
   devServerTask: ?Task,
   createNewProjectStart: () => any,
   showImportExistingProjectPrompt: () => any,
   clearConsole: (task: Task) => any,
+  showDeleteProjectPrompt: (project: any) => any,
 };
 
 class ApplicationMenu extends Component<Props> {
   menu: any;
 
   componentDidMount() {
-    this.buildMenu();
+    setTimeout(() => {
+      this.buildMenu();
+    }, 1000);
```

Comment:
I noticed this issue myself, during testing. I'll take a look after this lands :) @AWolf81 has some good ideas